package.json

```diff
@@ -4,11 +4,16 @@
   "description": "Flux Standard Action utlities for Redux",
   "main": "lib/index.js",
   "scripts": {
-    "test": "make test",
-    "prepublish": "make clean build"
+    "build": "npm run clean && babel src --out-dir lib --ignore *-test.js",
```

Comment:
The `test` folder is only to setup the test environment. Personally I like the approach of having a `__tests__` subfolder which is mirroring the structure of it's parent folder.

Another solution could be to remove the `init.js` and import `chai` in the test files.